sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/TaskService.java

```diff
@@ -37,11 +37,28 @@ public Task buildGeneratedTask() {
         .suggestedStartTime(LocalTime.of(11, 30))
         .dueDateDate(LocalDate.now().plusDays(1))
         .dueDateTime(LocalTime.of(11, 30))
-        .assignedTo("National USER - National User")
+        .assignedTo("Surveillance OFFICER - Surveillance Officer")
         .priority("Normal")
         .commentsOnTask(currentTimeMillis + "Comment on task")
         .commentsOnExecution(currentTimeMillis + "Comment on execution")
         .taskStatus("PENDING")
         .build();
   }
+
+  public Task buildEditTask() {
+    long currentTimeMillis = System.currentTimeMillis();
+    return Task.builder()
+        .taskContext("GENERAL")
+        .taskType("other task as described in comments")
+        .suggestedStartDate(LocalDate.now().plusDays(3))
+        .suggestedStartTime(LocalTime.of(12, 30))
+        .dueDateDate(LocalDate.now().plusDays(5))
```

Comment:
Maybe create a new public Constants class and define there + random methods when & where they are required from WebDriverHelpers, the sooner we do this the less refactoring we have to do.